imagetest/test_suites/storage_perf/iops_print_test.go

```diff
@@ -4,10 +4,131 @@
 package storageperf
 
 import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
 	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+const (
+	mkfsCmd           = "mkfs.ext4"
+	testreadOutputDir = "/mnt/disks/mount_dir"
 )
 
+type BlockDeviceList struct {
+	BlockDevices []BlockDevice `json:"blockdevices,omitempty"`
+}
+
+type BlockDevice struct {
+	Name string `json:"name,omitempty"`
+	Size string `json:"size,omitempty"`
+	Type string `json:"type,omitempty"`
+	// Other fields are not currently used.
+	X map[string]interface{} `json:"-"`
+}
+
+func getMountDiskPartition(diskExpectedSizeGb int) (string, error) {
+	lsblkCmd := "lsblk"
+	if !utils.CheckLinuxCmdExists(lsblkCmd) {
+		return "", fmt.Errorf("could not find lsblk")
+	}
+	lsblkout, err := exec.Command(lsblkCmd, "-o", "name,size,type", "--json").Output()
+	if err != nil {
+		return "", fmt.Errorf("failed to execute lsblk: %v", err)
+	}
+
+	var blockDevices BlockDeviceList
+	if err := json.Unmarshal(lsblkout, &blockDevices); err != nil {
+		return "", fmt.Errorf("failed to unmarshal lsblk output: %v", err)
+	}
+
+	diskExpectedSizeGbString := strconv.Itoa(diskExpectedSizeGb) + "G"
+	for _, blockDev := range blockDevices.BlockDevices {
+		if strings.ToLower(blockDev.Type) == "disk" && blockDev.Size == diskExpectedSizeGbString {
+			return blockDev.Name, nil
+		}
+	}
+
+	return "", fmt.Errorf("disk block with size not found")
+}
+
+func installFio() error {
+	var installFioCmd *exec.Cmd
+	if utils.CheckLinuxCmdExists("apt") {
+		installFioCmd = exec.Command("apt", "install", "-y", "fio")
```

Comment:
always run `apt update` before trying to install (there's no guarantee that a local package cache exists yet).

Reply:
changed to run update before install